src/TConsole.cpp

```diff
@@ -930,27 +912,13 @@ void TConsole::changeColors()
         mDisplayFont.setFixedPitch(true);
         mUpperPane->setFont(mDisplayFont);
         mLowerPane->setFont(mDisplayFont);
-        QPalette palette;
-        palette.setColor(QPalette::Text, mFgColor);
-        palette.setColor(QPalette::Highlight, QColor(55, 55, 255));
-        palette.setColor(QPalette::Base, QColor(Qt::black));
-        mUpperPane->setPalette(palette);
```

Comment:
All the drawing and highlighting is performed by us - so this and the similar stuff further down is pointless.